### ISSUE TITLE
Can i disable some of the recognized gestures ?

### ISSUE THREAD

**Issue by Trutifruti** [fabiopk/RT_GestureRecognition#3]
Hi me again! I was going through your code trying to figure out if I could disable some of the gestures? When I am testing, I get some mix-ups (notably between the two finger swipes and the regular ones). Do I have to retrain the model with only the move I want or can I just disable there recognition?

**Comment by fabiopk** [comment]
Hello Trutifruti!

Just a tip, I got better results showing the back of my hand to the camera when performing the two-finger swipes. But I understand that it is not perfect, so removing some gestures might make the final product more consistent. The best way to "disable" gestures without retraining the network would be to set their score to zero before choosing the best gesture.

I am not sure if you're using the `demo.py` as a base, but in any case, I'd say the quickest way would be to check the indexes of the gestures you want to ignore (probably checking `ges = label_dict[0].tolist()`) and modifying this line `out = (torch.nn.Softmax()(output).data).cpu().numpy()[0]` to set to zero (or even a negative number) the indexes of the gestures you want to disable BEFORE calling the torch.nn.Softmax(). So, for example:

```
# "Disable" gestures 
output[indexes_to_disable] = 0 # Or try a negative number
out = (torch.nn.Softmax()(output).data).cpu().numpy()[0]
```

Good luck!

**Comment by Trutifruti** [comment]
Awesome thanks i'll check that out !
